modules/alignment_processing.nf

```diff
@@ -50,68 +50,113 @@ process filter_soft_clipped_alignments {
   label 'samclipy'
   label 'smallTask'
 
-  publishDir "${params.output}/${params.tool}", mode: params.publish_dir_mode, pattern: "*.bam*"
+  publishDir (
+    path: "${params.output}/intermediate",
+    mode: params.publish_dir_mode,
+    pattern: "${name}*.bam{,.bai}",
+    overwrite: false,
```

Comment:
Why did you choose here `overwrite: false`? 🤔

Reply:
I disabled overwriting nearly everywhere to try to see if I'm accidentally overwriting files by placing them into the wrong directory with the saveAs functions. I'll remove that option when I'm confident I have that stuff working correctly.

Reply:
I removed all `overwrite: false` statements.